v03_pipeline/lib/misc/family_loading_failures_test.py

```diff
@@ -178,16 +178,16 @@ def test_all_relatedness_checks(self):
     def test_get_families_failed_sex_check(self):
         sex_check_ht = hl.Table.parallelize(
             [
-                {'s': 'ROS_006_18Y03226_D1', 'sex': 'M'},
-                {'s': 'ROS_006_18Y03227_D1', 'sex': 'F'},
-                {'s': 'ROS_006_18Y03228_D1', 'sex': 'F'},
-                {'s': 'ROS_007_19Y05919_D1', 'sex': 'F'},
-                {'s': 'ROS_007_19Y05939_D1', 'sex': 'F'},
-                {'s': 'ROS_007_19Y05987_D1', 'sex': 'F'},
+                {'s': 'ROS_006_18Y03226_D1', 'predicted_sex': 'M'},
+                {'s': 'ROS_006_18Y03227_D1', 'predicted_sex': 'F'},
+                {'s': 'ROS_006_18Y03228_D1', 'predicted_sex': 'F'},
+                {'s': 'ROS_007_19Y05919_D1', 'predicted_sex': 'F'},
+                {'s': 'ROS_007_19Y05939_D1', 'predicted_sex': 'F'},
+                {'s': 'ROS_007_19Y05987_D1', 'predicted_sex': 'F'},
```

Comment:
could test for an 'F' inappropriately called 'M' : maybe just swap the sex for the last one , F->M instead of M->F

Reply:
good idea!